src/api/app/views/webui2/webui/project/show.html.haml

```diff
@@ -11,8 +11,8 @@
       .col-md-8
         %h3#project-title
           = @project.title.presence || @project
-          - if @project.url.present?
-            %small= link_to(@project.url, @project.url)
+        - if @project.url.present?
+          = link_to(@project.url, @project.url, class: 'mb-3 d-block')
```

Comment:
Does the link need to be a block element? We do the same somehow in package#show, but I don't understand why it's needed.